roles/etcd_migrate/tasks/migrate.yml

```diff
@@ -42,6 +44,7 @@
     --lease-duration 1h
   environment:
     ETCDCTL_API: 3
+    PATH: "/usr/local/bin:/var/usrlocal/bin:{{ ansible_env.PATH }}"
```

Comment:
sure that is not a blocker, we can go forward and merge this.  My observation was about setting `$PATH` instead of using `{{ openshift.common.admin_binary }}` which should already have the correct path to `oadm`.